### DISCUSSION TITLE
Platforms for Libre Hardware Manager

### DISCUSSION THREAD

**Discussion by scott967** [LibreHardwareMonitor/LibreHardwareMonitor#791] [Q&A]
I am developing an app to display hardware data from OHM/LHM.  I understand that LHM primarily supports Windows x32/x64.  I have seen some indication that OHM can run on Linux.  So besides Windows, x86 what other platforms can run LHM, in particular
ARM-based platforms
Linux on x86 or other
Android
iOS

it's ok if it needs something like wine, I just don't want to offer my app to a platform that can't run LHM.
TIA
scott967

**Comment by wiiznokes** [comment]
I tested it on Linux x64 and it works for some sensors but the majority don't. However there is no error when compiling and running.
I wish this library was cross platform..
If somebody have a good alternative for Linux, please let me know here.

**Reply by JohnTravolski** [reply]
Did you ever find an alternative?
